support-frontend/build.sbt

```diff
@@ -25,7 +25,7 @@ libraryDependencies ++= Seq(
   "com.gu.identity" %% "identity-auth-play" % "4.12",
   "com.okta.jwt" % "okta-jwt-verifier" % oktaJwtVerifierVersion,
   "com.okta.jwt" % "okta-jwt-verifier-impl" % oktaJwtVerifierVersion % Runtime,
-  "com.gu" %% "identity-test-users" % "0.8",
+  "com.gu" %% "identity-test-users" % "0.10.2-SNAPSHOT",
```

Comment:
tbc when this is released https://github.com/guardian/identity-test-users/pull/10